app/src/main/java/org/p2p/wallet/home/ui/main/HomePresenter.kt

```diff
@@ -127,7 +127,7 @@ class HomePresenter(
             ethereumInteractor.setup(userSeedPhrase = userSeedPhrase)
             PendingTransactionMergeWorker.scheduleWorker(context)
         } else {
-            Timber.e(IllegalStateException(), "ETH is not init, no seed phrase")
```

Comment:
For .w there is no entry in sentry, fyi

Reply:
.e is wrong here as well. 
If feature toggle is disabled we see red logs, just like app is not working